CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ include(AddQCWorkflow)
 # ---- Project ----
```

Comment:
this seems like a weird git artifact, can you rebase with master or avoid modifying this file in any other way?